include/libtorrent/ssl_stream.hpp

```diff
@@ -34,14 +34,14 @@ POSSIBILITY OF SUCH DAMAGE.
 #ifndef TORRENT_SSL_STREAM_HPP_INCLUDED
 #define TORRENT_SSL_STREAM_HPP_INCLUDED
 
-#include "libtorrent/config.hpp" // for TORRENT_USE_SSL
-
-#if TORRENT_USE_SSL
-
 #include "libtorrent/socket.hpp"
 #include "libtorrent/error_code.hpp"
 #include "libtorrent/io_context.hpp"
-#include "libtorrent/aux_/openssl.hpp"
+#include "libtorrent/ssl.hpp"
+
+#if TORRENT_USE_SSL
```

Comment:
actually, it's not clear to me why you moved this down. if `TORRENT_USE_SSL == 0`, this whole file will turn into nothing, right?

Reply:
Indeed, I just moved it down because it was shorter than including `config.hpp` beforehand.